external/webrtc-sdk.js

```diff
@@ -32691,9 +32691,17 @@ define("WebRTC_SDK/manager/RTCLocalMediaManager", [
       var deferred = Q.defer();
 
       var stream = this.model.get("localAudioStream");
+      if (_.isNull(stream) || _.isUndefined(stream)) {
+        Logger.warn(
+          "Local Audio stream is null, only mute/unmute local backbone model."
+        );
+        this.model.set("localAudioMuted", mute);
+        deferred.resolve();
+        return deferred.promise;
+      }
```

Comment:
This is the bugfix: If stream does not exist (i.e. camera is unplugged) only update the backbone model - as opposed to updating the backbone model **and** muting the streams.